### ISSUE TITLE
Timed Out - Google Quota on large quality of calendar events

### ISSUE THREAD

**Issue by BearBritches** [derekantrican/GAS-ICS-Sync#39]
I'm encountering a "Timed Out" error when running this script on a manually imported calendar which has 541 calendar events. The error is presented at roughly 1800s 
![Screen Shot 2019-04-09 at 3 00 03 PM](https://user-images.githubusercontent.com/7284075/55829943-e7847280-5add-11e9-9082-257f327b1fff.png)

**Comment by binarypickle** [comment]
Same error for syncing a large calendar. Looks like we're hitting the quota on per execution runtime per https://developers.google.com/apps-script/guides/services/quotas.

Don't know if it's fully fair to call it a bug if we're running up against Google limits, but definitely a challenge. 

Looking through the execution transcripts/timestamps, seems that there's something around the recurrence settings that is eating up a big chunk of time consistently. 

```
[19-06-06 11:43:23:435 PDT] RecurrenceRule.until([Fri May 17 14:00:00 PDT 2019]) [0.001 seconds]
[19-06-06 11:43:23:437 PDT] RecurrenceRule.setTimeZone([Central Standard Time]) [0.001 seconds]
[19-06-06 11:44:15:542 PDT] CalendarApp.newRecurrence() [0 seconds]
```

**Comment by dustinthurston** [comment]
I only have 115 events in the ics file and it's still timing out. My quota is 6 minutes. Why is the script taking so long to run? Seems like a bug unless this is running on a 1970's processor.

**Comment by BearBritches** [comment]
Use this one: https://script.google.com/d/1Nr6_7_BttOsc0S39VMF0m-qz3zupzHIyeKYKkOXPX8gnDu_gqTUH5B7i/edit?usp=sharing

**Comment by dustinthurston** [comment]
> Use this one: https://script.google.com/d/1Nr6_7_BttOsc0S39VMF0m-qz3zupzHIyeKYKkOXPX8gnDu_gqTUH5B7i/edit?usp=sharing

Thanks. It looks like it skips recurring meetings. At least it works. I assume I can run this on top of the google sync to still get my recurring meetings?

**Comment by derekantrican** [comment]
### Closing as this is fixed by the latest release: [v5.0](https://github.com/derekantrican/GAS-ICS-Sync/releases/tag/5.0)

*If you find that there are still bugs, please comment on this issue and let us know*
